lib/dfe/analytics/load_entity_batch.rb

```diff
@@ -6,6 +6,11 @@ class LoadEntityBatch < AnalyticsJob
       BQ_BATCH_MAX_BYTES = 10_000_000
 
       def perform(model_class_arg, ids, entity_tag)
+        unless DfE::Analytics.enabled?
```

Comment:
Maybe we don't need at all if we put the guard in import_entities.rake - It only gets called from the rake task right ?